### PR TITLE
fix(ui): Hide the mission description pane if no mission is selected (#10123)

### PR DESCRIPTION
**Pull request by nickshanks**
**Content (Artwork / Missions / Jobs)**
images/ui/mission empty.png

This PR addresses the bug/feature described in issue #10123

## Summary
I didn't like the big empty box, so I deleted it.

## Screenshots
See original ticket for before & after screenshots.

## Testing Done
I landed on Dagon Station and compared that to landing on Luna.

## Save File
Any save file with more than a few minutes of exploration will work.

## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [X] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-assets/pull/167
 - [X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/407

## Performance Impact
N/A

### CODE REVIEW

File: data/_ui/interfaces.txt [hunk 1]
```diff
@@ -1293,8 +1293,13 @@ interface "bank"
 
 
 interface "mission" bottom
+	visible if "has description"
 	sprite "ui/mission"
 		align bottom
+	visible if "!has description"
+	sprite "ui/mission empty"
+		align bottom
+	visible
```
Comment (TomGoodIdea) [inline data/_ui/interfaces.txt:1303]:
```suggestion

	visible
```
looks better

**Review by RisingLeaf — APPROVED** [review]

File: data/_ui/interfaces.txt (hunk 1 above)
Comment (nickshanks) [reply data/_ui/interfaces.txt:1303]: Am I expected to click "Commit suggestion" or is this something that will be done if/when the PR is merged?

Comment (TomGoodIdea) [reply data/_ui/interfaces.txt:1303]: In this repo usually PR authors commit review suggestions themselves (if they agree to the proposed changes). And, as Amazinite said in #10123, PRs are squash merged, so you don't need to worry about your commit messages.

Comment (nickshanks) [reply data/_ui/interfaces.txt]: I had missed that reply, thanks. Done now, on both PRs.

**Review by TomGoodIdea — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

**Pull request merged**